game.cpp

```diff
@@ -1134,6 +1134,23 @@ void test_game() //!OCLINT tests may be many
   }
 #endif
 
+//#define FIX_ISSUE_463
+#ifdef FIX_ISSUE_463
+  // Players lose invulnerability after a short time
+  {
+    game g;
+    player& p = g.get_player(0);
```

Comment:
@richelbilderbeek is using a ref here a good idea?
It saves a lot of retyping `g.get_player(0)` below, but I'm unsure if this helps or hinder readability.

Reply:
@TheoPannetier: If it increases readablity, yes, you can opt to do so :-)